### PR TITLE
Added new Avi module to setup User Roles.

### PR DESCRIPTION
**Pull request by grastogi23**
##### ISSUE TYPE

 - New Module Pull Request

##### COMPONENT NAME

network.avi.avi_role

##### ANSIBLE VERSION

```
ansible 2.3.0
  config file = /home/grastogi/.ansible.cfg
  configured module search path = Default w/o overrides
```

##### SUMMARY

New module to setup user roles.

```

```

### CODE REVIEW

**Comment by khaltore** [comment]
shipit

**Comment by ericsysmin** [comment]
shipit

File: lib/ansible/modules/network/avi/avi_role.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            - Avi controller URL of the object.
+    uuid:
+        description:
+            - Unique object identifier of the object.
+extends_documentation_fragment:
+    - avi
+'''
+
+EXAMPLES = """
+- name: Example Adds/Deletes Role configuration from Avi Controller
```
Comment (gundalow) [inline lib/ansible/modules/network/avi/avi_role.py]: This appears to add, rather than delete

**Review by gundalow — CHANGES_REQUESTED** [review]

**Comment by ericsysmin** [comment]
shipit

**Review by gundalow — APPROVED** [review]

**Pull request merged**